<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cell_forgetPW</class>
 <widget class="QDialog" name="cell_forgetPW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>400</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>400</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>167</width>
      <height>63</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_login">
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>61</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>71</width>
         <height>61</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/loginIMG/image/forgetPWD.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ledit_phone">
            <property name="minimumSize">
             <size>
              <width>171</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>1000</width>
              <height>1000</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>请输入正确手机号码</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_3">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ledit_PW">
            <property name="minimumSize">
             <size>
              <width>171</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>10000</width>
              <height>10000</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>请输入修改的密码</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>219</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pbtn_confirm">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #4CAF50; /* 设置按钮背景颜色 */
    color: white; /* 设置文字颜色 */
    border: 1px solid #4CAF50; /* 设置按钮边框 */
    border-radius: 5px; /* 设置按钮圆角 */
    padding: 5px 10px; /* 设置按钮内边距 */
    font-size: 14px; /* 设置文字大小 */
}

QPushButton:hover {
    background-color: #45a049; /* 鼠标悬停时的背景颜色 */
}

QPushButton:pressed {
    background-color: #3e8e41; /* 鼠标按下时的背景颜色 */
}</string>
       </property>
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtn_backToLogin">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #4CAF50; /* 设置按钮背景颜色 */
    color: white; /* 设置文字颜色 */
    border: 1px solid #4CAF50; /* 设置按钮边框 */
    border-radius: 5px; /* 设置按钮圆角 */
    padding: 5px 10px; /* 设置按钮内边距 */
    font-size: 14px; /* 设置文字大小 */
}

QPushButton:hover {
    background-color: #45a049; /* 鼠标悬停时的背景颜色 */
}

QPushButton:pressed {
    background-color: #3e8e41; /* 鼠标按下时的背景颜色 */
}</string>
       </property>
       <property name="text">
        <string>返回登录界面</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
